mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb

```diff
@@ -127,7 +127,6 @@ func TestAccContainerCluster_misc(t *testing.T) {
 }
 
 func TestAccContainerCluster_withAddons(t *testing.T) {
-	t.Skipf("Skipping test %s due to https://github.com/hashicorp/terraform-provider-google/issues/16114", t.Name())
```

Comment:
drive-by: I think you want to revert the commit that created this change- you probably started from a dirty branch rather than `main`?